sdk/identity/Azure.Identity/tests/ClientAssertionCredentialLiveTests.cs

```diff
@@ -33,7 +33,12 @@ public async Task AuthnenticateWithAssertionCallback(bool useAsyncCallback)
         {
             var tenantId = TestEnvironment.ServicePrincipalTenantId;
             var clientId = TestEnvironment.ServicePrincipalClientId;
+
+#if NET9_0_OR_GREATER
```

Comment:
[SYSLIB0057: X509Certificate2 and X509Certificate constructors for binary and file content are obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0057) - load certificates using APIs that depend on the intended content type, instead of using the obsoleted APIs that allow multiple formats.